tests/Unit/Common/Exceptions/ServiceExceptionTest.php

```diff
@@ -111,7 +111,7 @@ public function testGetResponse()
     public function testNoWarningForNonXmlErrorMessage()
     {
         // Warnings are silenced in parseErrorMessage once they are converted to exceptions
-        \PHPUnit_Framework_Error_Warning::$enabled = false;
+        \PHPUnit\Framework\Error\Warning::$enabled = false;
```

Comment:
This line will report an error: `Fatal error: Class 'PHPUnit\Framework\Error\Warning' not found in C:\Program Files (x86)\Jenkins\workspace\PHP-full-build-manual\tests\Unit\Common\Exceptions\ServiceExceptionTest.php on line 114`